blog/2024-03-08-release-24-03.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Tractus-X 24.03 available now
+description: Release Tractus-X 24.03 announcement 
+slug: new-release-2403
+date: 2024-03-08T11:00
+hide_table_of_contents: false
+authors:
+  - name: Maximilian Ong
+    title: Ecosystem & Kits
+    url: https://github.com/maximilianong
+    image_url: https://github.com/maximilianong.png
+    email: maximilian.ong@catena-x.net
+---
+
+![Release Image](@site/static/img/Tractus-X_Release24.03.png)
+
+We are pleased to announce our first minor release in 2024!
+
+Tractus-X Release 24.03 focuses on improving the performance and stability of our Catena-X core and onboard services, strengthening our data space foundation.
+
+Use Cases & Value Creation: With five new KITs we are introducing exciting new offerings to solve the toughest industry problems such as company certificate management, data governance, manufacturing as a service, environmental & social standards (ESS).
+
+The Industry Core is an integrative layer between the industry-agnostic dataspace foundation and the use cases. It ensures that the various applications can access common functionalities. It describes how digital twins of components, materials and vehicles can be created for the automotive industry and linked with direct business partners. 
+
+Data Space Governance: The Catena-X Automotive Network e.V. will provide the corresponding standards (https://catena-x.net/de/standard-library).
```

Comment:
Isn't the Markdown linter complaining here? I would have expected the URL to look like this <https://catena-x.net/de/standard-library>